spec-update-draft.md

```diff
@@ -1622,26 +1627,52 @@ Some of these control criteria correspond to similar controls from at least thre
 Where relevant, corresponding controls from those frameworks are identified and linked from ValOS controls.
 
 ### Controls for Access Control
+@@@@
```

Comment:
What are those doing here? I am not sure if I am not aware of some Github markdown functionality, but I thought I'd point them out.

Reply:
yep, will do in an editorial cleanup after this round of merges - i.e. today